### PR TITLE
Acquia Search Upgrade

### PR DESCRIPTION
**Pull request by ben-hamelin**
## Summary

This upgrades Acquia Search and all related modules/config as needed to get the sites working with Solr 7.
Changes the install profile to only install Acquia Connector at first, since testing to date has required a 2-step process to get everything working. Once connector is installed, then installing Acquia search seems to work.

Introduces two new optional "features" which are basically the database search and solr search config. Using this approach we can provide both search options, with database being there as a fallback in case Solr isn't working on a site for some reason.

Updates to some theme and config to align with the updated machine names for the view and blocks for Solr search.

## Metadata

| Question | Answer |
|----------|--------|
| Did you use a meaningful pull request title? | yes
| Did you apply meaningful labels to the pull request? | yes
| Did you perform a self review first? | yes
| Documentation reflects changes? | no
| `CHANGELOG` reflects changes? | yes
| Unit/Functional tests reflect changes? | no
| Did you perform browser testing? | yes
| Risk level |  Medium
| Relevant links | https://thinkoomph.jira.com/browse/RIGA-35

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,12 +10,17 @@ modified Semantic Versioning scheme. See the "Versioning scheme" section of the
 
 ## [Unreleased]
 ### Added
+- RIGA-10: Added paragraph translation sync settings to person and location features.
```
Comment (jordan-caldwell) [inline CHANGELOG.md:13]: Is this included in this PR? I didn't spot it

**Review by jordan-caldwell — APPROVED** [review]
This looks good. Had one question of minor importance.

File: CHANGELOG.md (hunk 1 above)
Comment (ben-hamelin) [reply CHANGELOG.md]: Nope, messed up the rebase. Thanks!

**Comment by jordan-caldwell** [comment]
Looks good!

**Pull request merged**